assets/l10n/en_US.json

```diff
@@ -18,6 +18,7 @@
     "general.delete.all": "Delete all",
     "general.copy.success": "Copied to clipboard",
     "general.copy.clickToCopy": "Click to copy",
+    "general.confirm.okay": "Okay",
```

Comment:
I don't think this is necessary, use `done` or `confirm` instead

Reply:
I've removed this localization message as well as the entire dialog. I followed your advice on using `Flow.of(context)` instead so the dialog is no longer needed.